tests/runtime/http_server/test_http_server.py

```diff
@@ -544,6 +545,80 @@ def test_health_check_ok(runtime_http_server):
         assert response.text == "OK"
 
 
+def test_runtime_info_ok(runtime_http_server):
+    """Make sure the runtime info returns version data"""
+    with TestClient(runtime_http_server.app) as client:
+        response = client.get(http_server.RUNTIME_INFO_ENDPOINT)
+        assert response.status_code == 200
+
+        json_response = json.loads(response.content.decode(response.default_encoding))
+        assert "caikit" in json_response
+        assert "runtime_image" not in json_response
+        # dependent libraries not added if sys_modules not set to true
+        assert "py_to_proto" not in json_response
+
+
+def test_runtime_info_ok_response_all_sys_modules(runtime_http_server):
+    with temp_config(
+        {
+            "runtime": {
+                "versioning": {
+                    "sys_modules": True,
+                    "runtime_image": "1.2.3",
```

Comment:
This is the config details I have so far, but open to other thoughts @gabe-l-hart ! I know we talked about user requested packages in a list vs all sys modules vs just caikit. Current logic is just caikit libraries if nothing is set and adds in `runtime_image` and all sys.modules if set

Reply:
Ok, makes sense! A couple of thoughts:

1. We need to add default values to `caikit/config/config.yml`
2. I think it might be good to change `"versioning"` to `"version_info"` to avoid any confusion about things like "how to handle multiple service interface versions" in the future
3. I still see `"foo": "bar"` in here which I think should get removed?
4. I think we should change `"sys_modules"` to `"all_packages"` or something similar to avoid confusion with "standard modules"

Reply:
1,2,3 is good feedback, thanks! `foo: bar` is in as a test example that if other values are added in, they are not included in the final versioning response/dict.